rpcq/_client.py

```diff
@@ -56,18 +57,21 @@ def __init__(self, endpoint: str, timeout: float = None):
         # Cache of replies so that different tasks can share results with each other
         self._replies: Dict[str, Union[RPCReply, RPCError]] = {}
 
-    async def call_async(self, method_name: str, *args, **kwargs):
+    async def call_async(self, method_name: str, timeout: float = None, *args, **kwargs):
```

Comment:
I'm still unsure about reserving `timeout` because that it's a reasonably common kwarg that I'd like to be able to pass to an rpc method.

Reply:
Discussed offline: we should go with `rpcq_timeout`